newIDE/app/src/InstancesEditor/index.js

```diff
@@ -804,26 +833,38 @@ export default class InstancesEditor extends Component<Props> {
       this.highlightedInstance.setInstance(null);
   };
 
+  _onUpInstance = (
+    instance: gdInitialInstance,
+    sceneX: number,
+    sceneY: number
+  ) => {
+    if (!this.hasCursorMovedSinceItIsDown) {
+      this.props.instancesSelection.selectInstance({
+        instance,
+        multiSelect: this.keyboardShortcuts.shouldMultiSelect(),
+        layersLocks: this._getLayersLocks(),
+      });
+      if (this.props.onInstancesSelected) {
+        this.props.onInstancesSelected(
+          this.props.instancesSelection.getSelectedInstances()
+        );
+      }
+
+      if (this.selectionRectangle.hasStartedSelectionRectangle()) {
+        this._selectInstanceInsideRectangle();
+      }
+    }
+  };
+
   _onMoveInstance = (
     instance: gdInitialInstance,
     deltaX: number,
     deltaY: number
   ) => {
     this.fpsLimiter.notifyInteractionHappened();
-    if (
-      !this.hasCursorMovedSinceItIsDown &&
-      this.keyboardShortcuts.shouldCloneInstances()
-    ) {
-      this.hasCursorMovedSinceItIsDown = true;
-
-      const selectedInstances = this.props.instancesSelection.getSelectedInstances();
-      for (var i = 0; i < selectedInstances.length; i++) {
-        const instance = selectedInstances[i];
-        this.props.initialInstances
-          .insertInitialInstance(instance)
-          .resetPersistentUuid();
-      }
-    }
+
+    const hasCursorMovedSinceItIsDown = this.hasCursorMovedSinceItIsDown;
```

Comment:
maybe naming this variable slightly differently would help understand the nuance between both variables?
Something like `const hadCursorMoved = this.hasCursorMovedSinceItIsDown;` ?